sdk/python/runtime/main.go

```diff
@@ -73,7 +43,7 @@ func (m *PythonSdk) Codegen(modSource *Directory, subPath string, introspectionJ
 	})
 
 	modified := ctr.Directory(ModSourceDirPath)
-	diff := modSource.Diff(modified)
+	diff := modSource.Diff(modified).Directory(subPath)
```

Comment:
Looks good but can we add integ test coverage for this? Surprised there wasn't one for a python module w/ a subpath

Reply:
Yep! I'll take the opportunity to split into a separate PR though.